.github/workflows/slash_command_dispatch.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           token: ${{ secrets.PAT }}
           commands: |
             benchmark
+            benchmark_master
```

Comment:
I don't think this does anything and it's not needed too. Master won't be triggered by slash command